integration-cli/docker_cli_proxy_test.go

```diff
@@ -10,6 +10,9 @@ import (
 func TestCliProxyDisableProxyUnixSock(t *testing.T) {
 	cmd := exec.Command(dockerBinary, "info")
 	cmd.Env = []string{"HTTP_PROXY=http://127.0.0.1:9999"}
+	if testHost := os.Getenv("DOCKER_TEST_HOST"); testHost != "" { // preserve remote test daemon addr
```

Comment:
This should just be passing `DOCKER_HOST` along directly.  See #10797 where I've fixed the other instance of this. :+1: